scapy/contrib/gtp_v2.py

```diff
@@ -542,6 +545,56 @@ class IE_BearerFlags(gtp.IE_Base):
                    BitField("PPC", 0, 1)]
 
 
+class IE_MMContext_EPS(gtp.IE_Base):
+    name = "IE MM Context (EPS Security Context and Quadruplets)"
+    fields_desc = [ByteEnumField("ietype", 107, IEType),
+                   FieldLenField("length", None, length_of="length",
+                                 adjust=lambda pkt, x: len(pkt.payload) +
+                                 4, fmt="H"),
+                   BitField("CR_flag", 0, 4),
```

Comment:
Could you replace this 4 bits BitField with a ByteField?

Reply:
I can; but i was trying to follow the same syntax from other IEs.
Everywhere in this module, we use CR_flag & instance as 4 bit field.